bundles/vi/insert_map.moon

```diff
@@ -27,22 +33,11 @@ insert_map = {
       blink_interval: -> config.cursor_blink_interval
   }
 
-  escape: (editor) ->
-    state.insert_edit = get_edit editor
-    insert_pos = nil
-    state.change_mode editor, 'command'
-    editor.cursor.column = math.max 1, editor.cursor.column - 1
-
-  'ctrl_[': (editor) ->
-    state.insert_edit = get_edit editor
-    insert_pos = nil
-    state.change_mode editor, 'command'
-    editor.cursor.column = math.max 1, editor.cursor.column - 1
-
+  escape: (editor) -> exit_insert_mode editor
```

Comment:
I think this can just be written as

`escape: exit_insert_mode`

Same for next line.

Otherwise looks good to me!

Reply:
Alright, I've done so.

Reply:
I've found that the ctrl + [ doesn't work when writing python code at least, there's probably something overriding the key combo for that syntax, I'll investigate further at some point.

Reply:
Do you still want to merge this then?